boards/rapidsilicon/virgo_tc_eval_soc_proto/Kconfig.virgo_tc_eval_soc_proto

```
@@ -0,0 +1,5 @@
+# Copyright (c) 2021 Andes Technology Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_VIRGO_TC_EVAL_SOC_PROTO
```

```suggestion
config BOARD_VIRGO_EVAL_PROTO
```
Perhaps this name? Lets talk about this if you disagree

---

Let us give the name of the folder to board name. VIRGO_PROTO